### PR TITLE
Split Intel AI Tools CPU and GPU images.

### PR DESCRIPTION
**Pull request by sharvil10**
This PR splits the Intel AI Tools images into CPU and GPU images for Intel TensorFlow and Intel PyTorch.

## Description

This PR will split the Intel AI Tools CPU & GPU images into separate images. Th exact changes are  described below.

1. Split Intel PyTorch into Intel PyTorch CPU & XPU images.
2. Split Intel TensorFlow into Intel TensorFlow CPU & XPU images.
3. Change base image of Intel Jupyter images to be the ubi9 base images instead of the runtime images.

## How Has This Been Tested?

This was tested by running make commands to build containers, deploy K8 resources and testing images.

## Merge criteria:

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by sharvil10** [comment]
OpenShift Release PR: https://github.com/openshift/release/pull/51822

File: intel/base/gpu/ubi9-python-3.9/Dockerfile [hunk 1]
```diff
@@ -11,13 +11,13 @@ LABEL name="odh-notebook-intel-base-gpu-ubi9-python-3.9" \
...
 
-USER 0
+USER root
 WORKDIR /opt/app-root/src
 
 RUN . /etc/os-release && \
     dnf install -y 'dnf-command(config-manager)' && \
     dnf config-manager --add-repo \
-        https://repositories.intel.com/gpu/rhel/${VERSION_ID}/lts/2350/unified/intel-gpu-${VERSION_ID}.repo  && \
+        https://repositories.intel.com/gpu/rhel/9.3/lts/2350/unified/intel-gpu-9.3.repo  && \
```
Comment (atheo89) [inline intel/base/gpu/ubi9-python-3.9/Dockerfile:20]: Add this as reference here https://github.com/opendatahub-io/notebooks/pull/519
We will close this pr as you move this change here

**Comment by sharvil10** [comment]
Seems like it failed because the env variables were wrong for the jupyter images. I fixed it in this PR on [OpenShift Release CI #51853](https://github.com/openshift/release/pull/51853).

**Comment by sharvil10** [comment]
Is it okay to request more resources(CPU and Memory) in the statefulset of jupyter containers to test them? Also, the tests seem to fail arbitrarily locally as well. It works sometimes and sometimes it doesn't with the same error as seen here.

**Comment by harshad16** [comment]
@sharvil10  Is it okay to request more resources(CPU and Memory) in the statefulset of jupyter containers to test them? Also, the tests seem to fail arbitrarily locally as well. It works sometimes and sometimes it doesn't with the same error as seen here.

In the Opendatahub, user would have the option to pick different resource limits, 
if the question about  the resource limits in the testings, we would have to look it up.

Was there a reason to close this PR ?

**Pull request closed without merging**